.gitignore

```diff
@@ -41,6 +41,8 @@ binary-url.json
 .eslintrc.js
 cli/visual-snapshots
 
+# Undo some liberal git merge file ignores
+!*.base.js
```

Comment:
can this be removed?

Reply:
The original one is *.BASE.* which overrides .releaserc.base.js

Reply:
So I can add `.releaserc.base.js` with `-f` but i figured we want support for stuff like `webpack.config.base.js`

Reply:
If git was case-sensitive it would be fine... but it's not.

Reply:
wow, let's just remove all of those `# Created by git when using merge tools for conflicts` entries. I dont see a need for them

Reply:
Yeah? Mmkay